gnuradio-runtime/lib/pmt/pmt_serialize.cc

```diff
@@ -279,7 +279,10 @@ bool serialize(pmt_t obj, std::streambuf& sb)
     }
 
     if (is_pair(obj)) {
-        ok = serialize_untagged_u8(PST_PAIR, sb);
+        if (is_dict(obj))
```

Comment:
```suggestion
        if (is_dict(obj)) {
```

Reply:
Not sure what you are getting at here; most of the code here does not use brackets for one-line if statements but i can certainly add here if its desired.

Reply:
What about this?
```c++
ok = serialize_untagged_u8(is_dict(obj) ? PST_DICT : PST_PAIR, sb);
```

Reply:
thats cleaner, thanks; updated.